utils/slice.go

```diff
@@ -31,3 +31,21 @@ func ConvertToStringSlice(a interface{}) []string {
 	}
 	return result
 }
+
+func DeduplicateStringSlice(in []string) (out []string) {
+	for _, i := range in {
+		if !StringSliceContains(i, out) {
+			out = append(out, i)
+		}
+	}
+	return out
+}
+
+func StringSliceContains(needle string, haystack []string) bool {
```

Comment:
This is the same as https://github.com/Velocidex/velociraptor/blob/5d0a345d607901b06c0fec97df7c300abb9c5a8f/utils/utils.go#L36 but probably better named :-)

Reply:
Ah good find 😄 I did not see that one, I'll remove the duplicate code.